### ISSUE TITLE
Dockerizing imongo

### ISSUE THREAD

**Issue by suredream** [gusutabopb/imongo#8]

**Comment by gusutabopb** [comment]
No, I haven't ever thought about it, but it sound like a good idea.

I have rather limited experience with Docker, but I believe you'd have to pack a Jupyter, MongoDB and the IMongo kernel in a single Docker image.

I don't have time to work on this at the moment, but PRs are welcome!
